Devops Content Update (#1454)
kamranahmedse/developer-roadmap: content/roadmaps/102-devops/content/102-managing-servers/101-live-in-terminal/126-cat.md

> # Cat
> Cat(concatenate) command is very frequently used in Linux. It reads data from the file and gives their content as output. It helps us to create, view, concatenate files.
> 
> 
> **Syntax:**
> 
> View : `$ cat [option] [input-file]`
> 
> create : `$ cat [content] > [new-file]`
> 
> append : `$ cat [append_content] >> [existing-file]`
> 
> e.g. *$ cat file.txt*
> <ResourceGroupTitle>Free Content</ResourceGroupTitle>
> <BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.tecmint.com/13-basic-cat-command-examples-in-linux/'>Cat Command with examples</BadgeLink>
> <BadgeLink colorScheme='yellow' badgeText='Wiki' href='https://en.wikipedia.org/wiki/Cat_(Unix)'>Options</BadgeLink>

Reviewer: ```suggestion
# cat

`cat` (concatenate) command is very frequently used in Linux. It reads data from the file and gives its content as output. It helps us to create, view, and concatenate files.

It has the following syntax:

* View : `$ cat [option] [input-file]`
* Create : `$ cat [content] > [new-file]`
* Append : `$ cat [append_content] >> [existing-file]`

e.g. `$ cat file.txt`

<ResourceGroupTitle>Free Content</ResourceGroupTitle>
<BadgeLink colorScheme='yellow' badgeText='Read' href='https://www.tecmint.com/13-basic-cat-command-examples-in-linux/'>Cat Command with examples</BadgeLink>
<BadgeLink colorScheme='yellow' badgeText='Wiki' href='https://en.wikipedia.org/wiki/Cat_(Unix)'>Options</BadgeLink>
```
